### PR TITLE
Add raw total counts to M2 autoval tables

### PR DESCRIPTION
**Pull request by davidbenjamin**
As requested by Alyssa Macbeth.  @LeeTL1220 can you review?

### CODE REVIEW

File: scripts/mutect2_wdl/unsupported/calculate_sensitivity.py [hunk 1]
```diff
@@ -82,9 +82,16 @@ def assign_indel_af_bin(af):
                               columns = flatten([(str(x) + "_LCI", str(x) + "_UCI") for x in af_bins]))
 
     sensitivity_with_confidence = sensitivity_table.merge(error_bars, left_index = True, right_index = True)
-    sensitivity_with_confidence.to_csv( snp_or_indel + "_sensitivity.tsv", sep = '\t')
 
-    return sensitivity_with_confidence, af_bins
+    # put in the raw number of calls
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/unsupported/calculate_sensitivity.py]: Can I have a little bit of a higher level description of what you are doing?  Both in this code here and in general?  I don't know there specifics of what was requested...

**Comment by LeeTL1220** [comment]
@davidbenjamin Back to you.  Just need a quick description of what you are doing in the new code block.

**Comment by LeeTL1220** [comment]
@davidbenjamin All good!  Go ahead and merge

**Comment by davidbenjamin** [comment]
@LeeTL1220 FYI I just addressed your "and in general" with another commit.

**Pull request merged**